Mathlib/ModelTheory/Basic.lean

```diff
@@ -29,16 +29,10 @@ structures.
 * A `FirstOrder.Language.Embedding`, denoted `M ↪[L] N`, is an embedding from the `L`-structure `M`
   to the `L`-structure `N` that commutes with the interpretations of functions, and which preserves
   the interpretations of relations in both directions.
-* A `FirstOrder.Language.ElementaryEmbedding`, denoted `M ↪ₑ[L] N`, is an embedding from the
```

Comment:
Where did this bullet point move? Or why is it dropped?

Reply:
`ElementaryEmbedding` is defined in `ModelTheory.ElementaryMaps`, not in `ModelTheory.Basic`.

Reply:
The corresponding explanation is already in `ModelTheory.ElementaryMaps`.

Reply:
Thanks :+1: